desktop/lib/messagebundles/it.po

```diff
@@ -2896,28 +2871,24 @@ msgstr ""
 "aiuto e aiutare gli altri."
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/limegroup/gnutella/gui/StatusLine.java:208
-#, fuzzy
 msgid "Circle FrostWire on G+"
-msgstr "Usa FrostWire per..."
+msgstr "FrostWire cerchio su G +"
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/limegroup/gnutella/gui/StatusLine.java:223
-#, fuzzy
 msgid "<b>Seeding</b><p>completed torrent downloads.</p>"
-msgstr "<b>Seeding</b><p>download torrent completati."
+msgstr "<b>Seminando</b><p>download torrent completati."
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/limegroup/gnutella/gui/StatusLine.java:223
-#, fuzzy
 msgid ""
 "<b>Not Seeding</b><p>File chunks might be shared only during a torrent "
 "download.</p>"
 msgstr ""
-"<b>Non in Seeding</b>.<p>Porzioni di file possono essere condivise solo "
+"<b>Non seminando</b>.<p>Porzioni di file possono essere condivise solo "
 "durante<p>un download di torrent."
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/limegroup/gnutella/gui/StatusLine.java:468
-#, fuzzy
 msgid "Not Seeding"
-msgstr "Distribuzione seme"
+msgstr "Non seminando"
```

Comment:
`Non Seminando` (capital S)

Reply:
Capital letters work differently than in English. "No se está compartiendo", no debería ser "No se está Compartiendo"